### DISCUSSION TITLE
Doubt: Are we able to disable server components completely?

### DISCUSSION THREAD

**Discussion by akhilmhdh** [vercel/next.js#66909] [App Router]
I love the App router idea and router changes it brought. Enabling easier context split etc. 

But i don't want to use Server Side rendering rather just client side build. Is there a way to disable Server components completely, instead of putting `use client` everywhere? 

Mainly something like [Static Export](https://nextjs.org/docs/app/building-your-application/deploying/static-exports) Yes, but want everything to be read as  client side component on development too.

**Comment by icyJoseph** [comment]
Hi,

`use client` is still SSR'd:

- https://github.com/reactwg/server-components/discussions/4

You can defer rendering to the client though:

- https://demystifying-rsc.vercel.app/client-components/no-ssr/
- https://demystifying-rsc.vercel.app/client-components/next-dynamic/

And you can also use `useSearchParams` to cause a suspense fallback trigger, but this one is less ideal, I guess.
